Description: drop extra =back from POD
 To see the error this fixes, run pod2man lib/IO/Socket/SSL.pod | man -l -
 .
 It complains about an =item without =over around line 1493
Author: Damyan Ivanov <dmn@debian.org>
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=127133
Bug: https://rt.cpan.org/Ticket/Display.html?id=127133

--- a/lib/IO/Socket/SSL.pod
+++ b/lib/IO/Socket/SSL.pod
@@ -1488,8 +1488,6 @@ parent process.
 For more details and other arguments see C<stop_SSL> which gets called from
 C<close> to shutdown the SSL state of the socket.
 
-=back
-
 =item B<sysread( BUF, LEN, [ OFFSET ] )>
 
 This function behaves from the outside the same as B<sysread> in other
